### PR TITLE
chore(AIOT-225): python 3 code modernization

### PR DESCRIPTION
**Pull request by patocarroll**
getting rid of str.format() and replacing with fstrings

### CODE REVIEW

File: smartobjects/restitution/__init__.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def __getitem__(self, item):
         if not isinstance(item, int):
             raise IndexError("item must be an integer")
         if not 0 <= item < len(self._rows):
-            raise IndexError("Invalid index: must be in the range [{}, {})".format(0, len(self._rows)))
+            raise IndexError(f"Invalid index: must be in the range [{0}, {len(self._rows)})")
```
Comment (mvonthron) [inline smartobjects/restitution/__init__.py:30]:
```suggestion
            raise IndexError(f"Invalid index: must be in the range [{0}, {len(self._rows)-1})")
```

File: smartobjects/restitution/__init__.py [hunk 2]
```diff
@@ -133,7 +134,7 @@ def get(self, item, rtype=None):
             raise ValueError("Invalid type for argument 'item'")
 
         if not 0 <= index < len(self._source):
-            raise IndexError("Invalid index: must be in the range [{}, {})".format(0, len(self._source)))
+            raise IndexError(f"Invalid index: must be in the range [{0}, {len(self._source)})")
```
Comment (mvonthron) [inline smartobjects/restitution/__init__.py:137]: should probably be
```suggestion
            raise IndexError(f"Invalid index: must be in the range [{0}, {len(self._source)-1})")
```

Comment (patocarroll) [reply smartobjects/restitution/__init__.py:137]: Why the "-1", wasn't there before. You think it was a mistake before as well?

Comment (mvonthron) [reply smartobjects/restitution/__init__.py:137]: Nope it's my bad. I didn't pay attention to the brackets, they are indeed correctly written: `[x, y)` (and not  `[x, y]`)

**Pull request merged**